ui/packages/consul-ui/app/components/consul/discovery-chain/resolver-card/index.hbs

```diff
@@ -51,7 +51,7 @@
               <dt
                 {{tooltip (concat child.Failover.Type ' failover')}}
               >
-                {{concat child.Failover.Type ' failover'}}
+                {{child.Failover.Type}} failover
```

Comment:
All of this stuff just makes things consistent, no reason why redirect was capitalized and failover wasn't - also now we have DC and/or Partition redirects I've added the type of redirect to the tooltip.